test/uk/gov/hmrc/agentaccesscontrol/service/AuthorisationServiceSpec.scala

```diff
@@ -351,39 +351,11 @@ class AuthorisationServiceSpec extends UnitSpec with MockitoSugar {
 
   "isAuthorisedForAfi" when {
 
-    "agent suspension feature flag is OFF" should {
-
-      val agentSuspensionFeatureFlag = false
-
-      "return false when agent suspension feature flag is off and relationships exist" in new Context {
-        whenAgentSuspensionFeatureFlagIsChecked thenReturn agentSuspensionFeatureFlag
-
-        afiRelationshipConnectorIsCheckedForRelatioinships thenReturn Future(
-          true)
-
-        await(
-          authorisationService
-            .isAuthorisedForAfi(agentCode, nino, afiAuthDetails)) shouldBe true
-      }
-
-      "return false when agent suspension feature flag is off and relationships do not exist" in new Context {
-        whenAgentSuspensionFeatureFlagIsChecked thenReturn agentSuspensionFeatureFlag
-
-        afiRelationshipConnectorIsCheckedForRelatioinships thenReturn Future(
-          false)
-
-        await(
-          authorisationService
-            .isAuthorisedForAfi(agentCode, nino, afiAuthDetails)) shouldBe false
-      }
-    }
-
     "agent suspension feature flag is ON" should {
 
-      val agentSuspensionFeatureFlag = true
 
       "return false when error encountered fetching agent record from DES" in new Context {
-        whenAgentSuspensionFeatureFlagIsChecked thenReturn agentSuspensionFeatureFlag
+        whenAgentSuspensionFeatureFlagIsChecked thenReturn true
```

Comment:
I think you should be able to remove this line? (and the def) `whenAgentSuspensionFeatureFlagIsChecked` wouldn't be expected to happen anymore... if the test fails without it there'll be something else to change